frontend/src/features/properties/components/forms/subforms/InformationForm.tsx

```diff
@@ -47,7 +47,7 @@ const InformationForm: FunctionComponent<InformationFormProps> = (props: Informa
   /** only SRES can change to Disposed  */
   const classifications = keycloak.hasClaim(Claims.ADMIN_PROPERTIES)
     ? props.classifications
-    : props.classifications.filter(x => x.value !== Classifications.Disposed);
+    : props.classifications.filter(x => Number(x.value) !== Classifications.Disposed);
```

Comment:
Ha this kind of stuff isn't supposed to occur if we use typescript appropriately apparently, but looks like the `any` value is getting us

Reply:
I think we have the `SelectOption` value set up as `string | number` and happens to be string in this case haha

Reply:
But `any `has definitely got me before